docs/documentation/components/modal.html

```diff
@@ -119,15 +137,28 @@ <h3 class="title">Modal card</h3>
     <div class="content">
       <p>If you want a more classic modal, with a <strong>head</strong>, a <strong>body</strong> and a <strong>foot</strong>, use the <code>modal-card</code>.</p>
       <p>
-        <a class="button is-primary is-large modal-button" data-target="modal-ter">Launch modal card</a>
+        <button class="button is-primary is-large modal-button" data-target="modal-ter" aria-haspopup="true">Launch modal card</button>
       </p>
     </div>
 
-    <div class="highlight-full">
-      {% highlight html %}{{ modal_card }}{% endhighlight %}
-    </div>
-
-    {% include variables.html type='component' %}
+{% highlight html %}
```

Comment:
Edit the captured snippet, not the whole block here. And leave the variables partial